das-jira-connector/src/main/java/com/rawlabs/das/jira/tables/DASJiraTable.java

```diff
@@ -125,4 +126,43 @@ public OffsetDateTime getDateTime(String dateString) {
     // Parse the string to an OffsetDateTime object
     return OffsetDateTime.parse(dateString, formatter);
   }
+
+  // A helper to fallback to a default message if ever the body would be empty or null.
+  private static String mkMessage(String msg) {
+    if (msg == null || msg.trim().isEmpty()) {
+      return "Unknown JIRA API error";
+    }
+    return msg;
+  }
+
+  /**
+   * Helper method to create a DASSdk RuntimeException from an ApiException.
+   *
+   * <p>When the SDK exposes more exceptions (e.g. authentication errors), this method can
+   * investigate the ApiException and create the appropriate SDK exception.
+   *
+   * @param e the ApiException to convert
+   * @return the RuntimeException to throw
+   */
+  protected RuntimeException makeSdkException(ApiException e) {
+      return switch (e.getCode()) {
+          case 400 -> new DASSdkInvalidArgumentException(mkMessage(e.getResponseBody()), e);
+          case 401 -> new DASSdkInvalidArgumentException("unauthorized", e);
```

Comment:
I'm sure you thought the same, but should we now go an add these two to the DAS sdk, since we have them defined in the protocol?